Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -403,6 +403,37 @@ - (void)testLinkUser {
 
 }
 
+#pragma mark - Auth Credentials
+
+- (void)testAuthCredentials {
+    RLMAppCredentials *usernamePasswordCredential = [RLMAppCredentials credentialsWithUsername:@"test@mongodb.com" password:@"apassword"];
+    XCTAssertTrue([usernamePasswordCredential.provider isEqualToString:@"local-userpass"]);
```

Comment:
`XCTAssertEqualObjects(credential.provider, @"expected")` for all of these.

Reply:
done